Add `osu!mania Mapping World Cup 2026` contest article (#14473)
ppy/osu-wiki: wiki/Contests/MMWC/2026/en.md

| Judges       | ::{ flag=VN }:: [\[TCD\] Dzar03](https://osu.ppy.sh/users/16712231), ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773), ::{ flag=PH }:: [chocomilku-](https://osu.ppy.sh/users/16687756), ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699), ::{ flag=CN }:: [gzdongsheng](https://osu.ppy.sh/users/8660315), ::{ flag=GB }:: [Hydria](https://osu.ppy.sh/users/808176), ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640), ::{ flag=CA }:: [Ryax](https://osu.ppy.sh/users/8949769), ::{ flag=KR }:: [secXcscX](https://osu.ppy.sh/users/13543418), ::{ flag=US }:: [Unpredictable](https://osu.ppy.sh/users/7560872) |

## Links

- [Announcement news post]()
- [Forum post]()
- [Contest listing]()
- [Discord server](https://discord.gg/a7kpKK2rNj)
- [Contest spreadsheet]()
- [**Registration form**](https://docs.google.com/forms/d/e/1FAIpQLScpB8MP8-D4bHGuT1aR6_iq-cVprS0zrOmiEEbVQpJL9AyDug/viewform?usp=sharing&ouid=103196331030427249584)

## Ruleset

### Format

> - This contest is limited to the 4K keymode.

Reviewer: Clarified the gamemode alongside the keymode.
```suggestion
- This contest only accepts osu!mania 4K submissions.
```
Additionally, this may be better under "Rules" than "Format". This seems more like a submission rule rather than something exclusive to the format of the contest.

- Contest will be held on a team basis, each team consists of 3 or 4 mappers from the same country listed on their profile page.
- Signups may be done with a pre-formed team, or individually if you do not have a team. The organizers will assist in forming teams from the individual signup list based on their country or geographical location.
- There is no team limit per country  at the first stage. However, only a limited number of teams from each country will be allowed to advance to the later stages based on the contest proceedings. Therefore, all teams are free to participate initially.

> - There will be a total of 3 stages, each with its own specifications. Teams must create 1 map per stage according to the specifications.

Reviewer: Two things on this section:
- Could maybe be made into a table. Would make it easier to organise each round with its restriction name and an explanation of which.
- The format is using specific names for each stage, whereas the schedule further up the wiki article is using "numeric" stage names (Stage 1, Stage 2, etc...). I would personally suggest sticking with one naming format or the other to avoid any confusion. This more specific naming shown below seems to suit better, so I would change the schedule accordingly.

    - *Qualification Stage*: *Limited Drain Time*, where each team is required to create map with a specific duration limit.

>     - *Group Stage*: *Limited Artist*, where each team is required to create map using tracks by artists selected by the host. The available choices and selection order are influenced by team placement in the previous stage.

Reviewer: ```suggestion
  - *Group Stage*: *Limited Artist*, where each team is required to create map using tracks by artists selected by the host. The available choices and selection order are influenced by team placement in the previous stage.
```

    - *Final Stage*: *Limited Song Choice - Diverse Mapping Style*, where each team must create a map using a song selected from two songs provided by the host. Song selection order is determined by placement in the previous stage. Each mapper in the team should contribute a section with a distinct interpretation and mapping style.
- The difficulty range of the map will get harder as the stage progresses. Failure on meeting the difficulty requirement will affect the scoring.
- The judging criteria for each stage remain the same.

> 

Reviewer: ```suggestion
```

- **A mapper on the winning team must participate in at least 2 out of the 3 maps made to be eligible for the badge and points advantage.**
- **Submissions will not be accepted after the deadlines set for each Stage.**
- **All participants are expected to uphold sportsmanship and integrity throughout the contest.** Mutual respect toward other participants, observers, judges, and organizers is expected.
- **Sharing your work outside your team before judging concludes, or leaking confidential information, will lead into disqualification.** At the same time, it is also necessary to avoid being seen by others in editor mode as your current activities, thereby exposing song selection information.

### Judging criteria

- **Representation (30):** Points will be assigned based on how well the beatmap represents the song through its overall arrangement, patterns, and structure. As well as how well the map demonstrates the essence of collaboration mapping, which the beatmap could be diverse in execution but still packed as a one solid map.

> - **Creativity (25):** Points will be assigned based on how innovative the beatmap’s design is, including unique structural arrangements, logical intensity changes, and other technical aspects.

Reviewer: ```suggestion
- **Creativity (25):** Points will be assigned based on how innovative the beatmap's design is, including unique structural arrangements, logical intensity changes, and other technical aspects.
```

- **Playability (25):** Points will be assigned based on the extent to which the beatmap provides a cohesive playing, visual, and entertainment experience while maintaining a reasonable difficulty curve and fitting into the current stage's difficulty range.
- **Impression (20):** Points will be assigned based on the judges’ overall thought regarding the submission, as well as factors not covered by the three aspects above.

>   Scores were reported as standardised scores.

Reviewer: ```suggestion
Scores are reported as standardised scores.
```

| Designer | ::{ flag=ID }:: [Syadow-](https://osu.ppy.sh/users/3484485) |
| Statistician | ::{ flag=ID }:: [Reihynn](https://osu.ppy.sh/users/16630515) |
| Commentators | ::{ flag=SG }:: [Raveille](https://osu.ppy.sh/users/1388767), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
| Judges | ::{ flag=VN }:: [\[TCD\] Dzar03](https://osu.ppy.sh/users/16712231), ::{ flag=DO }:: [Antalf](https://osu.ppy.sh/users/8793773), ::{ flag=PH }:: [chocomilku-](https://osu.ppy.sh/users/16687756), ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699), ::{ flag=CN }:: [gzdongsheng](https://osu.ppy.sh/users/8660315), ::{ flag=GB }:: [Hydria](https://osu.ppy.sh/users/808176), ::{ flag=RU }:: [Kurisu Makise](https://osu.ppy.sh/users/2790640), ::{ flag=CA }:: [Ryax](https://osu.ppy.sh/users/8949769), ::{ flag=KR }:: [secXcscX](https://osu.ppy.sh/users/13543418), ::{ flag=US }:: [Unpredictable](https://osu.ppy.sh/users/7560872) |

## Links

- [Announcement news post]()
- [Forum post](https://osu.ppy.sh/community/forums/topics/2185948)
- [Contest listing]()

> - [Discord server](https://discord.gg/a7kpKK2rNj)

Reviewer 2: ```suggestion
- [Discord server](https://discord.com/invite/a7kpKK2rNj)
```
